_adr/76/index.adoc

```diff
@@ -0,0 +1,96 @@
+---
+num: 76
+title: "Handling usage of dynamically-async-created DNS records from dependent services"
+status: "Accepted"
```

Comment:
This was already accepted here: https://docs.google.com/document/d/1ypysP8u4eP7zvsYVScYi2SlRNwixXHrbazyiOoorQqA/edit

Is there an expected way to link the old location, for preservation of history (at least for those who have access to the old location)?

Reply:
No I don't think we need to preserve the old location. People can always look it up by the number.